source/js/components/paypal.js

```diff
@@ -13,9 +13,12 @@ export default function initPaypal(
   var loadingErrorMsg = window.gettext(
     "An error occurred. Please reload the page or try again later."
   );
-  var generalErrorMsg = window.gettext(
+  var authErrorMsg = window.gettext(
     "There was an error processing your payment. Please try again."
   );
+  var generalErrorMsg = window.gettext(
+    "Unable to connect to PayPal. Please try again in a few minutes."
```

Comment:
this phrasing doesn't feel very general: it implies that there are network/connectivity issues, not that the connection is perfectly fine but paypal doesn't like what it's being asked to do. Let's rephrase this just a little to something like "We were unable to establish a connection to PayPal. Please try again in a few minutes". Small change, but implies that it has nothing to do with the user's network conditions.

Reply:
Great point Pomax! I have updated the message and re-requested review